GoogleChromeLabs/ps-analysis-tool
# Feat: Add "Private Aggregation" panel

## Description

Private Aggregation panel under Private Advertising section

## Relevant Technical Choices

- Add details for "Private Aggregation" in `data/PSInfo.json` file.
- Create new component for Private Aggregation.

## Testing Instructions

- In the PSAT extension, open “Private Advertising” section and test if “Private Aggregation” section is added.
- Ensure it has content and all links are working as expected

## Additional Information:



## Screenshot/Screencast

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/2c1a17c6-6520-45fa-9302-0e25abed1c95">


---

## Checklist






Fixes #740
